### PR TITLE
Added a Jenkinsfile

### PR DESCRIPTION
**Pull request by emmberk**

### CODE REVIEW

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
build now

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        }
+        stage('Full Build') {
+            steps {
+                timeout(time: 10, unit: 'MINUTES') {
+                    sh './gradlew build'
+                }
+            }
+        }
+    }
+    post {
```
Comment (jhunzik) [inline Jenkinsfile]: ✏️ Can we get ✅ ❌ ⚠️ emojis in the response comments?

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
```json
Could not update commit status. Message: {
  "message": "Server Error"
}
```

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
This PR is just waiting on the status to successfully report back and complete the required check.

**Comment by jhunzik** [comment]
The `details` link for the build check status directs me to example.com. Did we want to change this?

**Comment by emmberk** [comment]
@jhunzik I know. I manually posted that status with an example URL. I'm still waiting for Jenkins to come back up to see if the status check changes I made in the Jenkins config work.

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
I'm still not sure why the status isn't working. I'm going to merge this for now to unblock the other PRs.

**Pull request merged**